docs/api.yaml

```diff
@@ -10950,6 +10950,10 @@ paths:
         Use this API to update one or all properties of an Entity. It will throw `400 - Bad Request` if any of the updating properties doesn't exist in the dataset.
 
         To unset value of any property, you can set it to empty string (""). Setting it to `null` will throw an error.
+
```

Comment:
Since you're editing the API docs here, would it make sense to also change the docs to show the new `conflict` and `conflictingProperties` flags on entities and entity versions? 

It's more to ask, but you could even start the changelog part of the docs?

Changed
* Entities accessed via API now have `conflict` and version-specific `conflictingProperties` fields
    * Conflicts are either hard (baseVersion conflicts and multiple versions update the same property), soft (baseVersion conflicts but data updates are independent), or null

Added
* Conflicts can be resolved using the entity PATCH endpoint with or without new data